DVP overview update (#15509)
docker/docker.github.io: docker-hub/publish/index.md

title: Docker Verified Publisher Program
redirect_from:
  - /docker-store/byol/
  - /docker-store/certify-images/
  - /docker-store/certify-plugins-logging/
  - /docker-store/customer_faq/
  - /docker-store/publisher_faq/
  - /docker-store/trustchain/
  - /docker-store/publish/
  - /docker-hub/publish/publish/
  - /docker-hub/publish/customer_faq/
  - /docker-hub/publish/publisher_faq/
  - /docker-hub/publish/certify-images/
  - /docker-hub/publish/certify-plugins-logging/
  - /docker-hub/publish/trustchain/
  - /docker-hub/publish/byol/
---

The Verified Publisher Program provides several features and benefits to Docker
Hub publishers. The following perks are granted based on participation tier:

- Verified publisher badge
- Insights and analytics
- Vulnerability scanning
- Additional Docker Business seats
- Removal of rate limiting for developers
- Co-marketing opportunities

## Verified publisher badge

The verified publisher badge signals high quality, and trust, to developers.
Images with this badge are verified as high quality, and the content can be
trusted.

![Docker, Inc. org with a verified publisher badge](./images/verified-publisher-badge.png)

> ## Insights and analytic

Reviewer: nit: Insights and analytics

description: Docker Verified Publisher Program
title: Docker Verified Publisher Program
redirect_from:
  - /docker-store/byol/
  - /docker-store/certify-images/
  - /docker-store/certify-plugins-logging/
  - /docker-store/customer_faq/
  - /docker-store/publisher_faq/
  - /docker-store/trustchain/
  - /docker-store/publish/
  - /docker-hub/publish/publish/
  - /docker-hub/publish/customer_faq/
  - /docker-hub/publish/publisher_faq/
  - /docker-hub/publish/certify-images/
  - /docker-hub/publish/certify-plugins-logging/
  - /docker-hub/publish/trustchain/
  - /docker-hub/publish/byol/
---

The Verified Publisher Program provides several benefits and features to Docker
Hub publishers. Including, but not limited to:

- Verified publisher badge
- Insights and analytics
- Vulnerability scanning
- Additional Docker Business seats

> - [Rate limiting](/docker-hub/download-rate-limit/){: target="blank"

Reviewer 2: Can we remove the 1 reference to Rate limiting and combine it in a note or parenthesis?

Author: Thanks for the suggestion. I rephrased the platform statement to clarify that the included perks depends on the participation tier. I think it works better.

  rel="noopener" class=""} exempt<sup>1</sup>

<sup>1</sup>: Depending on purchase plan.

## Verified publisher badge

> ![Docker, Inc. org with a verified publisher badge](./images/verified-publisher-badge.png)

Reviewer 2: Suggest moving the image after the content.

The verified publisher badge signals high quality, and trust, to developers.

> Images with this badge are verified to be of high quality, and that the content

Reviewer 2: ```suggestion
Images with this badge are verified as high quality, and the content
```

can be trusted.

## Insights and analytic

The [insights and analytics](/docker-hub/publish/insights-analytics){:
target="blank" rel="noopener" class=""} service provides usage metrics for how
the community uses your Docker images, and grants you insight into your user's
behavior.

![The insights and analytics tab on the Docker Hub website](./images/insights-and-analytics-tab.png)

> Select the time span for which you want to view analytics data, and export the

Reviewer 2: ```suggestion
Select the time span you want to view analytics data, and export the
```

description: Docker Verified Publisher Program
title: Docker Verified Publisher Program
redirect_from:
  - /docker-store/byol/
  - /docker-store/certify-images/
  - /docker-store/certify-plugins-logging/
  - /docker-store/customer_faq/
  - /docker-store/publisher_faq/
  - /docker-store/trustchain/
  - /docker-store/publish/
  - /docker-hub/publish/publish/
  - /docker-hub/publish/customer_faq/
  - /docker-hub/publish/publisher_faq/
  - /docker-hub/publish/certify-images/
  - /docker-hub/publish/certify-plugins-logging/
  - /docker-hub/publish/trustchain/
  - /docker-hub/publish/byol/
---

The Verified Publisher Program provides several benefits and features to Docker
Hub publishers, such as:

- Verified publisher badge
- Insights and analytics

> - Advanced image management

Reviewer 3: **question:** Is this a perk of DVPs? I thought it was something you got from a Team/Business plan 🤔

Author: I think this was a misunderstanding on my part, removed it.

> - Vulnerability scanning

Reviewer 3: **question:** same question as above, is this a DVP specific feature? 🤔

Author: Not specific to DVP, but seems like it is an included benefit. @Bkblodget what do you think about mentioning vulnerability scanning here as one of the benefits that a DVP publisher gets access to?

- No rate limiting

## Verified publisher badge

![Docker, Inc. org with a verified publisher badge](./images/verified-publisher-badge.png)

The verified publisher badge signals high quality, and trust, to developers.

> Images with this badge is verified to be of high quality, and that the content

Reviewer 3: **question:**
> Images with this badge is verified...

Not a native speaker so just wanted to confirm this sentence is correct 🤔 
Shouldn't it be "Images with this badge **are** verified..."?

Author: Totally right! Missed that!

per tag, and the raw format lists information about every image pull for the
selected time span. Data points include tag, type of pull, user geolocation,
client tool (user agent), and more.

## Advanced image management

[Advanced image management](/docker-hub/image-management/){: target="blank"
rel="noopener" class=""} is a tool that lets publishers browse and manage their
Docker Hub inventory. List, filter, and delete image tags in your repositories,
using the dashboard or API.

## Vulnerability scanning

Automatic vulnerability scanning using Snyk for images published to Docker Hub.
Scanning images ensures that the published content is secure, and underlines to
developers that it can be trusted. Scanning can be enabled on a per-repository
basis, refer to [vulnerability scanning](/docker-hub/vulnerability-scanning/){:
target="blank" rel="noopener" class=""} for more information about how to use
it.

## No rate limiting

For regular Docker images, image downloads are
[rate limited](/docker-hub/download-rate-limit/){: target="blank" rel="noopener"
class=""}. Images by verified publishers, however, are exempted from rate

> limiting; users can pull these images an unlimited number of times.

Reviewer 3: **issue:** I believe this depends on the tier of DVP. Not **all** DVPs get exempted from rate limiting, maybe we should be more specific about it here?

Author: I added a clarification that rate limit exempt depends on the purchasing plan.

## Who's eligible to become a verified publisher?

> Any independent software vendor who distribute software on Docker Hub can join

Reviewer 3: **question:**
>Any independent software vendor who distribute...

Shouldn't it be "distribute**s**"?

Author: Correct!
